0000-lts-releases.md

```diff
@@ -0,0 +1,54 @@
+- Start Date: 2022-04-20
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+The core team would like to move to an LTS release strategy
+
+# Motivation
+[motivation]: #motivation
+
+Our curret release strategy causes us to provide security and bugfixes for as
+many as five versions. Our documented [life cycle](https://saltproject.io/salt-platform-support/) has become less
```

Comment:
The link for _life cycle_ is now an HTML link redirecting to https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-supported-operating-systems.html